migration/src/test/java/ai/timefold/solver/migration/v8/ScoreGettersRecipeTest.java

```diff
@@ -71,130 +81,170 @@ void bendableLongScore() {
-                        "long softScore = score.softScore();\n" +
-                        "int initScore = score.initScore();\n");
+                """
+                        long hardScore = score.getHardScore();
+                        long softScore = score.getSoftScore();
+                        int initScore = score.getInitScore();
+                        """,
+                """
+                        long hardScore = score.hardScore();
+                        long softScore = score.softScore();
+                        int initScore = score.initScore();
+                        """);
     }
 
     @Test
     void simpleScore() {
         runTest("ai.timefold.solver.core.api.score.buildin.simple.SimpleScore",
                 "SimpleScore score = SimpleScore.of(1);",
-                "int value = score.getScore();\n" +
-                        "int initScore = score.getInitScore();\n",
-                "int value = score.score();\n" +
-                        "int initScore = score.initScore();\n");
+                """
+                        int value = score.getScore();
+                        int initScore = score.getInitScore();
+                        """,
+                """
+                        int value = score.score();
+                        int initScore = score.initScore();
+                        """);
     }
 
     @Test
     void simpleBigDecimalScore() {
         runTest("ai.timefold.solver.core.api.score.buildin.simplebigdecimal.SimpleBigDecimalScore",
                 "SimpleBigDecimalScore score = SimpleBigDecimalScore.of(BigDecimal.ONE);",
-                "BigDecimal value = score.getScore();\n" +
-                        "int initScore = score.getInitScore();\n",
-                "BigDecimal value = score.score();\n" +
-                        "int initScore = score.initScore();\n");
+                """
+                        BigDecimal value = score.getScore();
+                        int initScore = score.getInitScore();
+                        """,
+                """
+                        BigDecimal value = score.score();
+                        int initScore = score.initScore();
+                        """);
     }
 
     @Test
     void simpleLongScore() {
         runTest("ai.timefold.solver.core.api.score.buildin.simplelong.SimpleLongScore",
                 "SimpleLongScore score = SimpleLongScore.of(1L);",
-                "long value = score.getScore();\n" +
-                        "int initScore = score.getInitScore();\n",
-                "long value = score.score();\n" +
-                        "int initScore = score.initScore();\n");
+                """
+                        long value = score.getScore();
+                        int initScore = score.getInitScore();
+                        """,
+                """
+                        long value = score.score();
+                        int initScore = score.initScore();
+                        """);
```

Comment:
On our end we tend to use a small `.editorconfig` file to limit indentation here; not sure if that works with Spotless:
```
root = true

[*]
insert_final_newline = true
trim_trailing_whitespace = true

[src/test*/java/**.java]
indent_size = 4
ij_continuation_indent_size = 2
```